data/models/ships/lodos/lodos.model

```diff
@@ -14,5 +14,4 @@ anim gear_down 1 121
 collision lodos-col.dae
 
 lod 20
-mesh lodos-lo.dae
-
+mesh lodos-lo.dae
```

Comment:
No end of line here? Is this generated by blender (if so, I guess it's OK) or by hand (if so, then maybe a "RET")?

Reply:
It's by hand.